### ISSUE TITLE
lblMapName position initialized incorrectly in skirmish lobby

### ISSUE THREAD

**Issue by Crimsonum** [CnCNet/xna-cncnet-client#201]
The [lblMapName] element ignores the TextAnchor property when the skirmish lobby is first opened. It's positioned according to the Location flag (if the flag is not specified, uses the default value). In the default UI this isn't a problem, but in my custom UI I want the label to anchor to the center. This behaviour causes it to overlap with another label element.

Changing the map fixes it until the client is restarted. Multiplayer lobby works fine.

See the attached image for example.

The settings used in the example were:
```ini
[lblMapName]
TextAnchor=CENTER
AnchorPoint=806,317
```

![GIF 17 11 2020 13-11-20](https://user-images.githubusercontent.com/17765965/99383352-6a82c700-28d6-11eb-891b-c854150e3074.gif)

**Comment by Starkku** [comment]
Believe this has now been fixed in 6dc4de9e951a1311391974ac33e9452bfceade43. Prior to this commit, skirmish settings were loaded from INI before `SkirmishLobby` window and all controls were completely initialized. This is a change that was introduced in 2ebe432ff98a7d98b9e90359bac5d5c34b652eff due to errors with Discord status updates, but I believe the additional checks in `UpdateDiscordPresence` method that were put in place at same time made the change in initialization order unnecessary, which I have now reverted as it seems to have been a cause of problems such as this.

Will close once / if reported as fixed without further problems having been introduced.
